### ISSUE TITLE
Accessibility false positive aria-label invalid value

### ISSUE THREAD

**Issue by lancegliser** [storybookjs/storybook#17622]
**Describe the bug**
I'm using:

```html
<button aria-label="account of current user" />
```
The accessibility tools report a critical error of invalid value:

Critical
Invalid ARIA attribute value: aria-controls="menu-appbar-user"

<img width="499" alt="image" src="https://user-images.githubusercontent.com/12085479/156600522-3b929aeb-16cd-4dbd-95fd-2eec2959f444.png">

The spec for [aria-label](https://www.w3.org/TR/wai-aria-1.1/#aria-label) defines the type as `string` so any value should be valid.

**System**
Environment Info:

  System:
    OS: macOS 11.6.4
    CPU: (16) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
  Binaries:
    Node: 16.13.1 - /usr/local/bin/node
    Yarn: 1.22.10 - /usr/local/bin/yarn
    npm: 8.1.2 - /usr/local/bin/npm
  Browsers:
    Chrome: 98.0.4758.109
    Safari: 15.3
  npmPackages:
    @storybook/addon-a11y: ^6.4.5 => 6.4.14 
    @storybook/addon-actions: ^6.4.5 => 6.4.14 
    @storybook/addon-essentials: ^6.4.5 => 6.4.14 
    @storybook/addon-links: ^6.4.5 => 6.4.14 
    @storybook/react: ^6.4.5 => 6.4.14 

**Additional context**
Writing use the Material UI 4.x branch, but the tested output of HTML is above.

**Comment by certainlyakey** [comment]
Same with `aria-controls`: `aria-controls="panel-second"` gives an "Invalid ARIA attribute value" on Storybook and addon-a11y both on 6.5.9.

**Comment by valentinpalkovic** [comment]
Closing as obsolete. The axe-core version inside addon-a11y has been upgraded many times since 6.4.14 (8.5/8.6/9.0 rewrites of the addon), and the specific false positive on `aria-controls`/`aria-label` from axe-core 4.3 no longer reproduces. Material UI 4 is also long-deprecated. Please open a new issue if a current axe rule misfires on 10.x.
